copybook4java-codegen-maven-plugin/src/main/java/com/nordea/oss/copybook/codegen/CopyBookConverter.java

```diff
@@ -29,24 +29,25 @@ public CopyBookConverter() throws Exception {
         InputStream inputStream = this.getClass().getResourceAsStream("classconverter.html");
         String js = extractJS(inputStream);
 
-        manager = new ScriptEngineManager(null);
+        manager = new ScriptEngineManager();
```

Comment:
The null was added in https://github.com/NordeaOSS/copybook4java/pull/12 and seems to force loading the bundled Nashorn versions.  (https://stackoverflow.com/questions/25332640/getenginebynamenashorn-returns-null)

Also the first pull you made you added Nashorn as a dependency would that not be the better option so maybe the null could be removed also for older versions?

``` xml
<dependency>
  <groupId>org.openjdk.nashorn</groupId>
  <artifactId>nashorn-core</artifactId>
  <version>15.3</version>
</dependency>
```